util/libcrypto.num

```diff
@@ -266,7 +266,7 @@ WHIRLPOOL_Init                          271	3_0_0	EXIST::FUNCTION:DEPRECATEDIN_3
 EVP_OpenInit                            272	3_0_0	EXIST::FUNCTION:
 OCSP_response_get1_basic                273	3_0_0	EXIST::FUNCTION:OCSP
 CRYPTO_gcm128_tag                       274	3_0_0	EXIST::FUNCTION:
-OSSL_HTTP_parse_url                     275	3_0_0	EXIST::FUNCTION:HTTP
+OSSL_HTTP_parse_url                     275	3_0_0	EXIST::FUNCTION:
```

Comment:
This changed the wrong function - need to adapt `OSSL_parse_url` instead.

Reply:
:facepalm: Thanks!